content/channels/index.textile

```diff
@@ -1272,6 +1206,67 @@ The following features are not supported using subscription filters:
 * "Deltas":/channels/options/deltas
 * "Rewind ":/channels/options/rewind
 
+h3(#server-side-batching). Server-side batching
+
+Server-side batching optimizes message processing and delivery by grouping multiple messages into batches before sending them to subscribers. This feature reduces the overall message count, lowers costs, and mitigates the risk of hitting rate limits during high-throughput scenarios.
+
+Server-side batching is currently in beta status.
+
+h4. Key benefits
+
+* Cost efficiency.
+* Message rate limit mitigation.
+
+h4. Use cases
+
+The following use cases explain the benefits of server-side batching:
+
+* *High-volume chat applications*
+** Chat platforms experience large spikes in message volumes during live events like sports games or concerts. This scenario increases costs and risks hitting message rate limits.
+** Server-side batching groups multiple chat messages and reactions into single batches before delivery, reducing the number of individual messages processed and maintaining a seamless user experience.
+* *Fan engagement platforms*
+** Sports fan apps see surges in reactions during key moments, like goals, causing spikes in message volume.
+** Server-side batching combines multiple reactions into batches before sending them, managing high volumes efficiently and preventing rate limits from being exceeded.
+
+h4. How server-side batching works
+
+The server-side batching process works through a *batching policy* and a *batching interval:*
+
+h5. Batching policy
+
+The batching policy defines the rules and behaviors for aggregating messages during the batching interval. The supported *_simple_* policy groups multiple incoming messages into a single batch without altering their content or metadata.
+
+A batch can contain up to 200 messages by count or total data size. For example, if you have 210 messages, they will be split into two batches: one with 200 messages and another with 10. The batch size limit is 65,536 bytes. If the combined data size of 200 messages exceeds this limit, the excess bytes will be allocated to a new batch.
+
+h5. Batching interval
+
+The batching interval is a configurable period specified in milliseconds _ranging from 20ms to 1s_, during which incoming messages are collected and grouped into a batch. Messages sent to the server during this interval are temporarily held and aggregated. Once the interval elapses, the collected messages are combined into a single batch and delivered to subscribers as one message.
+
+h5. Choose the right interval for your app
+
+* Longer intervals are preferred for cost efficiency.
+* Align the interval with the expected user experience.
+
+h4. Configure server-side batching
+
+The following steps take you through the server-side configuration process:
+
+# On your "dashboard":ably.com/accounts/40946, select one of your apps.
```

Comment:
This URL isn't quite right for the dashboard and is pointing to a particular account. I think you want `https://ably.com/accounts/any`